src/dehydrated_devices.rs

```diff
@@ -0,0 +1,117 @@
+//! Dehydrated devices
+//!
+//! WASM wrapper for `matrix_sdk_crypto::dehydrated_devices`.
+
+use js_sys::{Array, JsString, Uint8Array};
+use matrix_sdk_crypto::dehydrated_devices;
+use wasm_bindgen::prelude::*;
+
+use crate::{identifiers::DeviceId, requests::PutDehydratedDeviceRequest, store::RoomKeyInfo};
+
+#[wasm_bindgen]
+#[derive(Debug)]
+/// Struct collecting methods to create and rehydrate dehydrated devices.
+pub struct DehydratedDevices {
```

Comment:
Was a bit surprised by the plural here, but it's the same as in the SDK.
It's more a Factory? or a Manager?

Reply:
Sort of a manager, but not quite.  Yeah, I'm not super happy with the naming, but it's what the SDK uses.